CMSIS/NN/Tests/UnitTest/TestCases/TestData/fully_connected/config_data.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2010-2020 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#pragma once
+// Generated by generate_test_data.py using TFL version 2.1.0 as reference.
+#define FULLY_CONNECTED_OUT_CH 6
+#define FULLY_CONNECTED_IN_CH 10
+#define FULLY_CONNECTED_INPUT_W 2
+#define FULLY_CONNECTED_INPUT_H 1
+#define FULLY_CONNECTED_FILTER_X 2
```

Comment:
filter x,y and output x, y are unused parameters. So, please remove them. We have a 2D input and a 2D filter.. arm_nnfunctions.h describe what parameters are used.

Reply:
Done